### ISSUE TITLE
Why no releases since Feb?

### ISSUE THREAD

**Issue by eherot** [test-kitchen/kitchen-ec2#65]
Lots of useful new features in master right now but the last release was in Feb.  What happened to continuous deployment?  Any plans to cut a release in the near future?

**Comment by willejs** [comment]
Sorry! @fnichol is there an ETA on a release?

**Comment by volker-fr** [comment]
I ran into the same problem and installed HEAD from the master branch to get some bugfixes I needed.

You can add it the following way (if you do not know how). Exchange `chef gem` with `gem` etc., depending on the environment you use.

```
cd /tmp
git clone https://github.com/test-kitchen/kitchen-ec2.git
cd kitchen-ec2
chef gem build kitchen-ec2.gemspec
ls kitchen-ec2-*.gem
# the version might differ, you get it from the ls output
chef gem install kitchen-ec2-0.8.1.dev.gem
```

**Comment by tyler-ball** [comment]
If you add a `Gemfile` to your cookbook with `gem "kitchen-ec2", path: "/Users/tball/github/kitchen-ec2"` (or `gem "kitchen-ec2", git: "https://github.com/test-kitchen/kitchen-ec2", branch: "master"`) you can bundle install the cookbook.

So locally I added a Gemfile to my test cookbook, `bundle install --binstubs`, `./bin/kitchen create`.

I'm also working on getting a release for kitchen-ec2 ready, but I want to refactor the code before I do that.  It may not happen until after ChefConf :disappointed:

**Comment by vitalis** [comment]
:+1:

**Comment by huntertj** [comment]
:+1:

**Comment by tyler-ball** [comment]
I just released kitchen-ec2 version 0.9.0 - please check it out!  I deprecated some configs (affected ones can be seen in the README) but they will continue to work until version 1.0.0.

My current roadmap is to release a 0.10.0 version which adds windows support.  Then I will release a 1.0.0 which removes the deprecated configs and pulls in any remaining bug fixes.

**Comment by vitalis** [comment]
:+1: Tank you @tyler-ball keep up the great work!!
